.github/workflows/msvc.yml

```diff
@@ -28,28 +28,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Create CMake project using SDL as a subproject
```

Comment:
Why don't you want to test this case?

Reply:
I originally disabled creating CPack packages when SDL is built as a suproject.
In the latest revision, this conditional has been moved to a new option `SDL_DISABLE_INSTALL_CPACK`.